deploy/charts/rook-ceph/templates/NOTES.txt

```diff
@@ -1,6 +1,13 @@
 The Rook Operator has been installed. Check its status by running:
   kubectl --namespace {{ .Release.Namespace }} get pods -l "app=rook-ceph-operator"
 
+{{- if eq .Values.csi.rookUseCSIOperator true }}
```

Comment:
I would suggest to remove lines 4-10 from these notes. The note on line 20 will be sufficient to point out that they are using the CSI operator. It's rare to need to check the status of the csi operator pod. Probably we should also remove line 2 above, no need to tell them to check the status of the rook operator either, but that can be a future PR.

Reply:
done